statsmodels/graphics/regressionplots.py

```diff
@@ -822,8 +822,6 @@ def plot_leverage_resid2(results, alpha=.05, label_kwargs={}, ax=None,
     alpha : float
         Specifies the cut-off for large-standardized residuals. Residuals
         are assumed to be distributed N(0, 1) with alpha=alpha.
-    label_kwargs : dict
-        The keywords to pass to annotate for the labels.
```

Comment:
I didn't see anything obvious and I could read the person who wrote it's mind.  Someone can always come back in and enhance it.